lib/internal/Magento/Framework/Code/Reader/ClassReader.php

```diff
@@ -8,6 +8,11 @@
 class ClassReader implements ClassReaderInterface
 {
     /**
+     * @var array
+     */
+    protected $_classNameParents;
```

Comment:
Please avoid using underscores in property names, it is a legacy standard.
There is no need for this property to be protected as we do not encourage inheritance-based APIs